entity-framework/core/get-started/netcore/new-db-sqlite.md

```diff
@@ -1,100 +1,91 @@
+- The [.NET Core SDK](https://www.microsoft.com/net/download/core) 1.1.x no longer supports `project.json` or Visual Studio 2015. We recommend you [migrate from project.json to csproj](https://docs.microsoft.com/dotnet/articles/core/migration/). If you are using Visual Studio, we recommend you migrate to [Visual Studio 2017](https://www.visualstudio.com/downloads/).
+- You can view this article's [sample](https://github.com/aspnet/EntityFramework.Docs/tree/master/samples/core/GetStarted/NetCore/ConsoleApp.SQLite) on GitHub.
 
 ## Prerequisites
 
 The following prerequisites are needed to complete this walkthrough:
 * An operating system that supports .NET Core.
-* [The .NET Core SDK](https://www.microsoft.com/net/core) provides the `dotnet` command line tool, which will be used to build and run your application.
-* A text editor or IDE of your choice.
+* [The .NET Core SDK](https://www.microsoft.com/net/core).
 
 ## Create a new project
 
-* Create a new `ConsoleApp.SQLite` folder for your project and use the `dotnet` command to populate it with a .NET Core application.
+* Create a new `ConsoleApp.SQLite` folder for your project and use the `dotnet` command to populate it with a .NET Core app.
 
 ```bash
 mkdir ConsoleApp.SQLite
 cd ConsoleApp.SQLite/
-dotnet new
+dotnet new console
 ```
 
 ## Install Entity Framework
 
 To use EF Core, install the package for the database provider(s) you want to target. This walkthrough uses SQLite. For a list of available providers see [Database Providers](../../providers/index.md).
 
-*  Modify the `ConsoleApp.SQLite.csproj` file so that it matches the following.
+*  Replace the `ConsoleApp.SQLite.csproj` contents with the following:
 
 [!code[Main](../../../../samples/core/GetStarted/NetCore/ConsoleApp.SQLite/ConsoleApp.SQLite.csproj)]
 
+ Note: The version numbers used above were correct at the time of publishing.
+
 *  Run `dotnet restore` to install the new packages.
 
-## Create your model
+## Create the model
 
-Now it's time to define a context and entity classes that make up your model.
+Define a context and entity classes that make up your model.
 
-* Create a new `Model.cs` file with the following contents.
+* Create a new *Model.cs* file with the following contents.
 
 [!code-csharp[Main](../../../../samples/core/GetStarted/NetCore/ConsoleApp.SQLite/Model.cs)]
 
-> [!TIP]
-> In a real application you would put each class in a separate file and put the connection string in a configuration file. For the sake of simplicity, we are putting everything in a single code file for this tutorial.
+Tip: In a real application you would put each class in a separate file and put the connection string in a configuration file. To keep the tutorial simple, we are putting everything in one file.
 
-## Create your database
+## Create the database
 
-Now that you have a model, you can use migrations to create a database for you.
+Once you have a model, you can use migrations to create a database.
```

Comment:
An introduction to migrations is missing -- or can you link to something for that?